eng/scripts/Create-APIView.ps1

```diff
@@ -138,6 +159,68 @@ function Invoke-SwaggerAPIViewParser {
     }
 }
 
+<#
+.DESCRIPTION
+  Invoke the TypeSpec parset to generate APIView tokens.
+
+.PARAMETER Type
+  New or Baseline TypeSpec APIView tokens.
+
+.PARAMETER ProjectPath
+  The TypeSpec Project path.
+
+.PARAMETER ResourceProvider
+  The ResourceProvider Name.
+
+.PARAMETER Tag
+  The Tag to use for generating the APIView Tokens.
+
+.PARAMETER TokenDirectory
+  The directory to store the generated APIView Tokens.
+
+.OUTPUTS
+  The resource provider name.
+#>
+function Invoke-TypeSpecAPIViewParser {
+  param (
+      [ValidateSet("New", "Baseline")]
+      [Parameter(Mandatory = $true)]
+      [string]$Type,
+      [Parameter(Mandatory = $true)]
+      [string]$ProjectPath,
+      [Parameter(Mandatory = $true)]
+      [string]$ResourceProvider,
+      [Parameter(Mandatory = $true)]
+      [string]$TokenDirectory
+  )
+  $tempWorkingDirectoryName = [guid]::NewGuid().ToString()
+  $tempWorkingDirectoryPath = [System.IO.Path]::Combine($TempDirectory, $tempWorkingDirectoryName)
+  New-Item -ItemType Directory -Path $tempWorkingDirectoryPath > $null
+
+  try {
+    Write-Host "Installing required dependencies to generate API review"
+    npm ci
+    Write-Host "List current npm package version"
+    npm list
+    Write-Host "Compiling files and generating '$Type' APIView for '$resourceProvider'..."
+    Push-Location $ProjectPath
+    npx tsp compile . --emit=@azure-tools/typespec-apiview --option @azure-tools/typespec-apiview.emitter-output-dir=$tempWorkingDirectoryPath/output/apiview.json
```

Comment:
```suggestion
    npm exec --no -- tsp compile . --emit=@azure-tools/typespec-apiview --option @azure-tools/typespec-apiview.emitter-output-dir=$tempWorkingDirectoryPath/output/apiview.json
```

Scripts should always use `npm exec --no -- package` instead of `npx package`, since it's the safest way to ensure the correct package and args are used.